game/scripts/npc/heroes/chatterjee.txt

```diff
@@ -82,9 +99,9 @@
     // Status
     //-------------------------------------------------------------------------------------------------------------
     "StatusHealth"                                        "200"
-    "StatusHealthRegen"                                   "1.0"
+    "StatusHealthRegen"                                   "0.5"
     "StatusMana"                                          "75"
-    "StatusManaRegen"                                     "0.0"
+    "StatusManaRegen"                                     "0.5"
```

Comment:
i just thought it'd be fun to split the starting regen between mana and health because weird chatterjee things